langchain/document_loaders/parsers/pdf.py

```diff
@@ -9,20 +9,37 @@
 class PyPDFParser(BaseBlobParser):
     """Loads a PDF with pypdf and chunks at character level."""
 
-    def lazy_parse(self, blob: Blob) -> Iterator[Document]:
+    def lazy_parse(
+        self,
+        blob: Blob,
+        start_page: Optional[int] = None,
```

Comment:
apologies for the delay!

we should make these init params not method params (since method is inherited, signature shouldn't be changed)